packages/design-system/src/components/Dropdown/DropdownMenuSection.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { DropdownMenuOption } from './DropdownMenuOption';
-import { ListState, Node } from 'react-stately';
-import { useListBoxSection } from 'react-aria';
+import { ListState, Node } from '../react-aria';
+import { useListBoxSection } from '../react-aria';
```

Comment:
could these lines be combined now that they're imported from the same place?

Reply:
Yeah, good call